spec/views/hq/orphans/_form_spec.rb

```diff
@@ -0,0 +1,103 @@
+require 'rails_helper'
+require 'cgi'
+
+RSpec.describe "hq/orphans/_form.html.erb", type: :view do
```

Comment:
We are also checking that select fields are selecting the correct option, text field have correct values ... in the Sponsor view we also check that some fields are not displayed in the form. I think it is easy to make mistakes here so better test it extensively.